crates/typst-library/src/layout/transform.rs

```diff
@@ -9,8 +9,9 @@ use crate::layout::{Abs, Alignment, Angle, HAlignment, Length, Ratio, Rel, VAlig
 ///
 /// # Example
 /// ```example
-/// #rect(inset: 0pt, move(
-///   dx: 6pt, dy: 6pt,
+/// >>> #set page(width: auto)
```

Comment:
I think making the page width auto might make the image scale up and look blurry in the docs.

Reply:
Okay, then I'll delete it.